modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -532,11 +532,11 @@
 		openWindow(config, callback) {
 			config.openingWindow = window;
 
-			var top = Util.getTop();
-
-			var topUtil = top.Liferay.Util;
+			var dialog = Window.getWindow(config);
```

Comment:
I think for this bit of code to work properly, the `liferay-util-window` module must already be loaded, which is why `_openWindowProvider` is provided in a `Liferay.provide` with a dependency listed [here](https://github.com/liferay-frontend/liferay-portal/pull/2205/files#diff-22ee0ee3757e1d50cbee0814c51e24fd2d987a91091b54dde287c3b7034ec30cL1403).

Reply:
`Liferay.Util.Window` is accessible either way, isn't that the "only" thing we care about here?

I mean, I've tested this and it works just as it did before the change.

Reply:
I see, so this needs to be done within the `Liferay.provide` deprecation project. Close the PR if you agree.